### PR TITLE
describing installation using pip

### PR DESCRIPTION
**Pull request by sepastian**
## Description

Describe installation using `pip` in `README.rst`.

## Motivation and Context

The current section on installation describes how to clone `pywb` and build/install it locally. 
This is complicated, it is enough to simply `pip install pywb`.

## Types of changes

- [ ] Replay fix (fixes a replay specific issue)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added or updated tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -58,7 +58,13 @@ Please see the `full documentation <https://pywb.readthedocs.org>`_ for more det
...
-To run and install locally you can:
+```shell
+pip install pywb
+```
+
+Note: depending on your Python installation, you may have to use `pip3` instead of `pip`.
+
+To install, test and build docs locally you can:
 
 * Install with ``python setup.py install``
```
Comment (edsu) [inline README.rst:69]: Is the indentation off here?

Comment (sepastian) [reply README.rst:69]: Not sure I see what you mean. It looks ok to me, the lines I inserted are flush with the left hand margin.

Comment (edsu) [reply README.rst:69]: My apologies, it must have been how I was looking at the diff!

**Review by edsu — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (sepastian) [reply README.rst:69]: No worries, thanks for the merge - and all the useful tools!

**Pull request merged**